src/daemon/remote.proto

```diff
@@ -5,8 +5,9 @@ package dist_clang.daemon.proto;
 
 // Sent from emitter to absorber.
 message Remote {
-  optional base.proto.Flags flags = 1;
-  optional string source          = 2;
+  optional base.proto.Flags flags        = 1;
+  optional string source                 = 2;
+  optional string sanitize_blacklist     = 3;
```

Comment:
Tried it, received

    daemon/remote.proto:14:3: Expected "required", "optional", or "repeated".
    daemon/remote.proto:14:6: Expected field name.

Seems that protobuf is indeed too old. I will add a `FIXME`